### DISCUSSION TITLE
Bottom-right osu! cookie aesthetics

### DISCUSSION THREAD

**Discussion by annoe1** [ppy/osu#17472] [Ideas]
The “osu!” Button in the bottom right is very bad looking. I wish it was not there. It’s big.

**Comment by frenzibyte** [comment]
is this forwarded to stable or lazer? this repository is for the new lazer client only, and the logo button looks much smaller on lazer compared to stable.

**Reply by annoe1** [reply]
This is for Lazer

**Reply by frenzibyte** [reply]
You may have the ability to customize its placement via skin editor eventually, but until then, I don't see how it can be reasonably smaller.

Out of curiosity, can you attach a screenshot of the full song select screen, to see how the osu! cookie button looks?

**Comment by smoogipoo** [comment]
Though I'm not sure about this, I believe flyte's redesigns don't have the osu! cookie anymore and we should probably make sure everyone's on board with this before anything gets implemented:
![image](https://user-images.githubusercontent.com/1329837/160741321-018bea83-df15-46e3-8308-ac72db614285.png)

**Reply by respektive** [reply]
Maybe the button could also be changeable in the skin-editor?

**Reply by peppy** [reply]
I've already told flyte multiple times that the osu! logo should be present on each screen, so he has some work to do still.
